kalite/settings/__init__.py

```diff
@@ -65,16 +69,16 @@ def package_selected(package_name):
 
 
 if package_selected("nalanda"):
-    LOG.info("Nalanda package selected")
+    warnings.warn(
+        "CONFIG_PACKAGE is outdated, use a settings module from kalite.project.settings",
+        RemovedInKALite_v015_Warning
+    )
     TURN_OFF_MOTIVATIONAL_FEATURES = True
     RESTRICTED_TEACHER_PERMISSIONS = True
     FIXED_BLOCK_EXERCISES = 5
     QUIZ_REPEATS = 3
 UNIT_POINTS = 2000
 
-# for extracting assessment item resources
-ASSESSMENT_ITEMS_ZIP_URL = "https://learningequality.org/downloads/ka-lite/%s/content/assessment.zip" % version.SHORTVERSION
```

Comment:
This is moved to a location further up the file so can be overwritten by other settings modules.